### ISSUE TITLE
tsd.core.timezone does not fix "now" button

### ISSUE THREAD

**Issue by mlosapio** [OpenTSDB/opentsdb#954]
The "now" button the GUI doesn't select the proper time. 

If metrics are collected in UTC and all systems are on UTC then I would expect setting tsd.core.timezone = UTC to make "now" time selector be the appropriate time in UTC. 

Is this not the case?

This issue was closed however I don't believe its correct: https://github.com/OpenTSDB/opentsdb/issues/258

**Comment by manolama** [comment]
Yeah this could use some work around detecting the browser timezone and using that. I don't know if we'll have a fix prior to 3.0 when we switch UIs.

**Comment by lucifercr07** [comment]
I am using opentsdb docker image. To host it on port 4242.

I am trying to send data with timestamp in milliseconds.
If I am using  timestamp = long((datetime.utcnow() - datetime(1970, 1, 1)).total_seconds() * 1000)
to generate timestamp I have to adjust the from and to time in browser to utc time, then only data is dislplayed. 

While using  timestamp = long((datetime.now() - datetime(1970, 1, 1)).total_seconds() * 1000) 
then shows it according to current time.

From what I know since time is being changed to utc milliseconds in first case there shouldn't have been any problem in having the metrics live in browser. While in second case it works fine. 

Why is it so?
